### PR TITLE
Non streaming for rolling batch

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — COMMENTED** [review]
please also change the python handler code. Enabling streaming will make the handler fall streaming path by default

**Review by lanking520 — APPROVED** [review]

**Review by lanking520 — CHANGES_REQUESTED** [review]
please fix the PMD issue

File: engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java [hunk 1]
```diff
@@ -48,11 +48,13 @@ class RollingBatch implements Runnable {
     private ReentrantLock lock;
     private Condition canAdd;
     private Condition canRead;
+    private boolean enableStreaming;
 
-    RollingBatch(PyProcess process, int maxRollingBatchSize, int timeout) {
+    RollingBatch(PyProcess process, int maxRollingBatchSize, int timeout, boolean enableStreaming) {
```
Comment (frankfliu) [inline engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java:53]: In which case we want to disable streaming if rolling batch is enabled?

Comment (lanking520) [reply engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java:53]: Some cases customer just want to get output in 1 go. Also for short sequence generation, not necessary for rolling batch

**Review by lanking520 — APPROVED** [review]

**Pull request merged**